templates/server/puppetserver/conf.d/auth.conf.erb

```diff
@@ -220,7 +220,20 @@ authorization: {
             sort-order: 500
             name: "puppetlabs report"
         },
+<%- if scope.function_versioncmp([@server_puppetserver_version, '5.5']) >= 0 -%>
```

Comment:
I guess this should be `6.0`?

Reply:
No, but 5.3. Seems like I misunderstood the Jira issue. The feature is available in the puppet-agent since 5.5, but in puppetserver since 5.3.